### ISSUE TITLE
Update/fix sample dependency to make audrey build with nightly

### ISSUE THREAD

**Issue by virtualritz** [RustAudio/audrey#27]
Audrey 0.2.0 depends on `sample 0.9.1`. This crate has been discontinued and is not called [dasp](https://crates.io/crates/dasp).

When using nightly Audrey breaks because of `sample`:
```
error[E0282]: type annotations needed
  --> /Users/moritz/.cargo/registry/src/github.com-1ecc6299db9ec823/sample-0.9.1/src/window.rs:73:41
   |
73 |         let v = phase.to_float_sample().to_sample() * PI_2;
   |                                         ^^^^^^^^^ cannot infer type for type parameter `S` declared on the associated function `to_sample`
   |
help: consider specifying the type argument in the method call
   |
73 |         let v = phase.to_float_sample().to_sample::<S>() * PI_2;
   |                                                  ^^^^^
```

This happens also on `master` which depends on `sample 0.10`.

`dasp` does build with `nightly`.

**Comment by est31** [comment]
PRs welcome!

**Comment by mitchmindtree** [comment]
Closed via #28.
